CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # developing
+
+- Env: non-empty GOBIN & GOPATH. [4613](https://github.com/beego/beego/pull/4613)
```

Comment:
you could put this anywhere so that you won't get conflicts.

Reply:
Got it.